PowerPointLabs/PowerPointLabs/ELearningLab/Converters/ShapeNameWidthConverter.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Text.RegularExpressions;
+using System.Windows;
+using System.Windows.Data;
+
+namespace PowerPointLabs.ELearningLab.Converters
+{
+    public class ShapeNameWidthConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
+        {
+            if (value is double)
+            {
+                return (double)value - 45;
```

Comment:
45 is a constant that is used multiple times, you can declare a constant locally.

Reply:
Replaced constant with a local constant